src/Sulu/Bundle/PreviewBundle/Resources/config/config_preview.yml

```diff
@@ -3,3 +3,6 @@ framework:
         storage_id: session.storage.mock_file
     profiler:
         enabled: false
+
+sulu_document_manager:
```

Comment:
Also I don't know how to easily test this class... Thought about unit testing the `PreviewKernel`, but that doesn't work, because it is inheriting from the global `WebsiteKernel`...